multinet/api.py

```diff
@@ -100,6 +102,23 @@ def get_table_rows(workspace: str, table: str, offset: int = 0, limit: int = 30)
     return Workspace(workspace).table(table).rows(offset, limit)
 
 
+@bp.route("/workspaces/<workspace>/tables/<table>/metadata", methods=["GET"])
+@require_reader
+@swag_from("swagger/get_metadata.yaml")
+def get_table_metadata(workspace: str, table: str) -> Any:
+    """Retrieve the metadata of a table, if it exists."""
+    metadata = Workspace(workspace).table(table).get_metadata()
```

Comment:
Will this return gracefully if the table/workspace is not found? How about if it is found but the metadata doesn't exist?

Reply:
1. If the table/workspace aren't found, the relevant 4XX errors will be raised and returned.
2. The `get_metadata` call returns `None` if the metadata doesn't exist. However, see my other comment with Roni regarding this.

Reply:
@JackWilb, going to resolve this; unresolve if you still have a problem here.